wiki/People/The_Team/Beatmap_Nominators/zh.md

```diff
@@ -0,0 +1,212 @@
+| ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | 中文, 粤语 |
+| ![][flag_SG] [neonat](https://osu.ppy.sh/users/1561995) | 中文 |
+| ![][flag_US] [radar](https://osu.ppy.sh/users/7131099) |  |
+| ![][flag_AT] [Stefan](https://osu.ppy.sh/users/626907) | 德语, 塞尔维亚语 |
+| ![][flag_JP] [tasuke912](https://osu.ppy.sh/users/2774767) | 日语 |
+| ![][flag_ID] [Volta](https://osu.ppy.sh/users/4154071) | 印度尼西亚语 |
+| ![][flag_CL] [Vulkin](https://osu.ppy.sh/users/4901066) | 西班牙语 |
+| ![][flag_FR] [Yales](https://osu.ppy.sh/users/2377881) | 法语, 日语 |
+
+#### osu!catch
+
+| 名字 | 语言 |
+| :-- | :-- |
+| ![][flag_TH] [-Luminate](https://osu.ppy.sh/users/4778689) | 泰语 |
+| ![][flag_PH] [Bunnrei](https://osu.ppy.sh/users/829284) | 菲律宾语 |
+| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) |  |
+| ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134) | 菲律宾语 |
+| ![][flag_RU] [Lacrima](https://osu.ppy.sh/users/4915649) | 俄语 |
+| ![][flag_GB] [MBomb](https://osu.ppy.sh/users/3071175) |  |
+| ![][flag_US] [wonjae](https://osu.ppy.sh/users/5032045) |  |
+| ![][flag_ID] [Xinely](https://osu.ppy.sh/users/1521445) | 印度尼西亚语, 部分中文 |
+
+#### osu!mania
+
+| 名字 | 语言 |
+| :-- | :-- |
+| ![][flag_MY] [\_Kobii](https://osu.ppy.sh/users/6209713) | 中文 |
+| ![][flag_CN] [\_Stan](https://osu.ppy.sh/users/1653229) | 中文 |
+| ![][flag_US] [Kamuy](https://osu.ppy.sh/users/7439226) | 部分韩语 |
+| ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | 韩语 |
+| ![][flag_AU] [Keiga](https://osu.ppy.sh/users/6866022) | 中文 |
+| ![][flag_HK] [PokeSky](https://osu.ppy.sh/users/3617111) | 中文, 粤语 |
+| ![][flag_ID] [Rivals_7](https://osu.ppy.sh/users/4610379) | 印度尼西亚语 |
+| ![][flag_DZ] [Scotty](https://osu.ppy.sh/users/11085809) | 阿拉伯语, 法语 |
+| ![][flag_US] [Unpredictable](https://osu.ppy.sh/users/7560872) |  |
+| ![][flag_ID] [Voxa](https://osu.ppy.sh/users/9229539) | 印度尼西亚语 |
+
+### 待定成员
+
+#### osu!standard
+
+| 名字 | 语言 |
+| :-- | :-- |
+| ![][flag_KR] [Affirmation](https://osu.ppy.sh/users/6186628) | 韩语 |
+| ![][flag_JP] [BaAR_Vendel](https://osu.ppy.sh/users/8679346) | 日语 |
+| ![][flag_RS] [MaridiuS](https://osu.ppy.sh/users/4496961) | 塞尔维亚语 |
+| ![][flag_US] [Nao Tomori](https://osu.ppy.sh/users/5364763) |  |
+| ![][flag_RU] [SMOKELIND](https://osu.ppy.sh/users/9327302) | 俄语 |
+| ![][flag_ID] [Uta](https://osu.ppy.sh/users/9439796) | 印度尼西亚语 |
+
+#### osu!taiko
+
+| 名字 | 语言 |
+| :-- | :-- |
+| ![][flag_ID] [Roger](https://osu.ppy.sh/users/1538223) | 印度尼西亚语, 部分日语 |
+
+#### osu!catch
+
+| 名字 | 语言 |
+| :-- | :-- |
+| ![][flag_AT] [Daletto](https://osu.ppy.sh/users/7592136) | 德语 |
+| ![][flag_ID] [Dapuluous](https://osu.ppy.sh/users/8140944) | 印度尼西亚语 |
+
+#### osu!mania
+
+| 名字 | 语言 |
+| :-- | :-- |
+| ![][flag_ID] [Amiichii](https://osu.ppy.sh/users/9228569) | 印度尼西亚语 |
+| ![][flag_KR] [Dubstek](https://osu.ppy.sh/users/9555243) | 韩语 |
+| ![][flag_NL] [UnluckyCroco](https://osu.ppy.sh/users/9461160) | 荷兰语 |
+
+[flag_AT]: /wiki/shared/flag/AT.gif "Austria"
```

Comment:
I think you should translate these (all pages)

Reply:
> I think you should translate country names (on all pages)

can I copy everything [here](https://github.com/ppy/osu-wiki/blob/master/meta/flag-references/en.md) directly, if I translate them?

Reply:
if you could make a zh.md in that folder that would be great. I didn't want to ask u to do that cuz there are a lot of lines lol